Source/WebCore/page/FrameView.h

```diff
@@ -977,6 +978,7 @@ class FrameView final : public ScrollView {
     ViewportRendererType m_viewportRendererType { ViewportRendererType::None };
     ScrollPinningBehavior m_scrollPinningBehavior { DoNotPin };
     SelectionRevealMode m_selectionRevealModeForFocusedElement { SelectionRevealMode::DoNotReveal };
+    ScrollAnchoringController m_scrollAnchoringController;
```

Comment:
Maybe this should be a unique_ptr<> and only allocated when scroll anchoring is enabled?

Reply:
Fixed.